docs_user/modules/proc_pulling-configuration-from-a-tripleo-deployment.adoc

```diff
@@ -22,7 +22,14 @@ remote_config_path=/tmp/tripleo
 +
 Make sure the ssh command you provide in `ssh_cmd` parameter is correct and with key authentication.
 
-. Enable or disable the services you want:
+. Enable or disable the services you want in /etc/os-diff/config.yaml:
+
+Make sure you have the correct rights to edit the file example:
```

Comment:
s/example/for example/ ? Alternatively the chown example could also be removed i think.